test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -643,6 +643,29 @@ Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
             '{"Microsoft.PowerShell:ExecutionPolicy": "RemoteSigned", "WindowsPowerShellCompatibilityNoClobberModuleList": ["' + $ModuleName2 + '"]}' | Out-File -Force $ConfigPath
             & $pwsh -NoProfile -NonInteractive -settingsFile $ConfigPath -c "[System.Management.Automation.Internal.InternalTestHooks]::SetTestHook('TestWindowsPowerShellPSHomeLocation', `'$basePath`');Test-${ModuleName2}PSEdition;Test-$ModuleName2" | Should -Be @('Desktop','Core')
         }
+
+        It "NoClobber WinCompat list in powershell.config is a Desktop-edition module" {
+            ## The 'RemoteDesktop' module (available on Windows Server) should not be imported twice.
```

Comment:
`RemoteDesktop` is available in both Windows Server 2016 and 2019. Our CIs are running in WinServer 2016 today and will be upgraded to 2019 at some point in future.
I'd like to a use a real module for validation here. If we find this test to be unstable because of that in CI/daily runs, I will update it using a test module.

Reply:
What about to run the test local on notebook? :-)

Reply:
Good call. I will add logic to use `PersistentMemory` on client.

Reply:
Done.